kolibri/plugins/device/assets/src/views/ManageContentPage/index.vue

```diff
@@ -136,11 +136,18 @@
         return '';
       },
       dropdownOptions() {
-        return [
-          { label: this.$tr('exportChannels'), value: 'EXPORT' },
-          { label: this.$tr('deleteChannels'), value: 'DELETE' },
-          { label: this.$tr('editChannelOrder'), value: 'REARRANGE' },
-        ];
+        if (this.installedChannelsWithResources.length > 1) {
+          return [
+            { label: this.$tr('exportChannels'), value: 'EXPORT' },
+            { label: this.$tr('deleteChannels'), value: 'DELETE' },
+            { label: this.$tr('editChannelOrder'), value: 'REARRANGE' },
+          ];
+        } else {
+          return [
+            { label: this.$tr('exportChannels'), value: 'EXPORT' },
+            { label: this.$tr('deleteChannels'), value: 'DELETE' },
+          ];
+        }
```

Comment:
A small nuance is that the expected behavior described in #6558 was "option to reorder should be disabled". (We generally prefer to disable rather than hide UI elements to make the experience more predictable.)

This is not tested, but you should be able to do something like:

```js
        return [
          { label: this.$tr('exportChannels'), value: 'EXPORT' },
          { label: this.$tr('deleteChannels'), value: 'DELETE' },
          {
            label: this.$tr('editChannelOrder'),
            value: 'REARRANGE',
            disabled: this.installedChannelsWithResources.length > 1,
          },
        ];
```